src/tt_importer.erl

```diff
@@ -0,0 +1,78 @@
+-module(tt_importer).
+-author("Adam Szlachta").
+
+-behaviour(gen_server).
+
+%% API
+-export([start_link/0, import_file/1]).
+
+%% gen_server callbacks
+-export([init/1, handle_call/3, handle_cast/2, handle_info/2, terminate/2, code_change/3]).
+
+-define(SERVER, ?MODULE).
+-define(TABLE, talks).
+
+% -record(talk,  {title :: string(), start_time :: calendar:datetime(), end_time :: calendar:datetime(), location :: string()}).
+
+
+%%%===================================================================
+%%% API
+%%%===================================================================
+
+-spec start_link() -> {ok, Pid :: pid()} | ignore | {error, Reason :: term()}.
+start_link() ->
+    gen_server:start_link({local, ?SERVER}, ?MODULE, [], []).
+
+-spec import_file(Filename :: string) -> ok.
```

Comment:
The type specs are like function invocations. So it should be `Filename :: string()`.